client/reducers/pings.js

```diff
@@ -1,10 +1,12 @@
 
-import {RECEIVE_PINGS} from '../actions/pings.js'
+import {RECEIVE_PINGS, POST_PING} from '../actions/pings.js'
 
 export default function pings (state = [], action) {
   switch (action.type) {
     case RECEIVE_PINGS:
       return action.pings
+    case POST_PING:
+      return action.ping
```

Comment:
This will overwrite the pings array with a single ping object, you want to add the single ping to the array